### ISSUE TITLE
Save Image Files After SfM Matching?

### ISSUE THREAD

**Issue by joshbg2k** [openMVG/openMVG#1562]
I have a circle with an array of cameras pointed toward the center of the circle, and I trigger all the cameras at the same time to capture a subject. What I want to do, in addition to 3D reconstruction, is create image files from each of the source images that are properly corrected. How can I use compute match and/or compute features tool data to correct the original image files?

**Comment by pmoulon** [comment]
What do you mean by corrected (can you elaborate)?
- Do you mean that you want to remove the lens distortion?

**Comment by joshbg2k** [comment]
Sure, let me first say that it's totally possible that I'm conflating problems here. My problem is that I have an array of, say, 24 cameras, and since it may be impossible to get every camera flawlessly positioned, I want to use software to tilt/skew/fix/etc, each image so that when I create a video from the image files, it's smooth and video frames are perfectly aligned. Does this make sense?

Now, from my testing it looks like the SfM tools are doing a pretty good job of making point clouds from some sample image sets I have, though I'm going to keep practicing. But I thought that I could somehow use the data from the matching tool to fix the variations in camera position in the image file to create a smooth video. Is this possible?

**Comment by pmoulon** [comment]
Yes, you could use the found camera orientation in order to interpolate the rotations between the frames.

**Comment by joshbg2k** [comment]
Thanks, I'll start playing around with this.
